src/main/java/org/killbill/billing/plugin/adyen/api/mapping/UserDataMappingService.java

```diff
@@ -69,8 +73,79 @@ public static UserData toUserData(@Nullable final Account account, final Iterabl
 
-        return userData;
-    }
+        //determine AccountAge Indicator
+        final String accountAgeIndicator = PluginProperties.findPluginPropertyValue(ACCOUNT_AGE_INDICATOR,properties);
+        final Optional<String> optionalAccountAgeIndicator = toLastName(accountAgeIndicator,account);
+        final String accAgeIndicator = optionalAccountAgeIndicator.isPresent() ? optionalAccountAgeIndicator.get() : null;
+        userData.setAccountAgeIndicator(accAgeIndicator);
+
+        final String accountChangeDate = PluginProperties.findPluginPropertyValue(ACCOUNT_CHANGE_DATE,properties);
+        if(accountChangeDate != null) {
+            userData.setAccountChangeDate(DateTime.parse(accountChangeDate));
+        }
+        final String accountChangeIndicator = PluginProperties.findPluginPropertyValue(ACCOUNT_CHANGE_INDICATOR,properties);
+        userData.setAccountChangeIndicator(accountChangeIndicator);
+
+        final String accountCreationDate = PluginProperties.findPluginPropertyValue(ACCOUNT_CREATION_DATE,properties);
+        if(accountCreationDate != null) {
+                userData.setAccountCreationDate(DateTime.parse(accountCreationDate));
+        }
+
+        final String passwordChangeDate = PluginProperties.findPluginPropertyValue(PASSWORD_CHANGE_DATE,properties);
+        if(passwordChangeDate != null) {
+            userData.setPasswordChangeDate(DateTime.parse(passwordChangeDate));
+        }
+        final String passwordChangeDateIndicator = PluginProperties.findPluginPropertyValue(PASSWORD_CHANGE_DATE_INDICATOR,properties);
+        userData.setPasswordChangeDateIndicator(passwordChangeDateIndicator);
+
+        final String purchasesLast6Months = PluginProperties.findPluginPropertyValue(PURCHASES_LAST_6_MONTHS,properties);
+        if(purchasesLast6Months != null) {
+            userData.setPurchasesLast6Months(Integer.valueOf(purchasesLast6Months));
+        }
+
+        final String addCardAttemptsDay = PluginProperties.findAndDecodePluginPropertyValue(ADD_CARD_ATTEMPTS_DAY,properties);
+        if(addCardAttemptsDay != null) {
+            userData.setAddCardAttemptsDay(Integer.valueOf(addCardAttemptsDay));
+        }
+
+        final String pastTransactionsDay = PluginProperties.findAndDecodePluginPropertyValue(PAST_TRANSACTIONS_DAY,properties);
+        if(pastTransactionsDay != null) {
+            userData.setPastTransactionsDay(Integer.valueOf(pastTransactionsDay));
+        }
+
+        final String pastTransactionsYear = PluginProperties.findAndDecodePluginPropertyValue(PAST_TRANSACTIONS_YEAR,properties);
+        if(pastTransactionsYear != null) {
+            userData.setPastTransactionsYear(Integer.valueOf(pastTransactionsYear));
+        }
+
+        final String paymentAccountAge = PluginProperties.findAndDecodePluginPropertyValue(PAYMENT_ACCOUNT_AGE,properties);
+        if(paymentAccountAge != null) {
+            userData.setPaymentAccountAge(DateTime.parse(paymentAccountAge));
```

Comment:
We should be able to backfill this from `AdyenPaymentMethodsRecord.getCreatedDate`